BootloaderCorePkg/Tools/GenerateKeys.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+## @ GenerateKeys.py
+#
+# This script is an reference to generate RSA private keys required by slimbootloader.
+# This should be one time key generation for a project based on usecase needs.and this would

```

Comment:
Remove the period between needs and and.

Reply:
done